include/tss2/tss2_tcti_tbs.h

```diff
@@ -0,0 +1,24 @@
+/* SPDX-License-Identifier: BSD-2 */
+/*
+ * Copyright (c) 2015 - 2018, Intel Corporation
```

Comment:
@davidjmaria you should talk to your legal department about this copyright statement since you / Facebook originated this code, not Intel.

Reply:
I talked to the legal department, I have now added the facebook copyright header but have left the intel copyright header in as well since this code is heavily based off of, and includes code from, the code for the device TCTI.